inc/plugins/auth_local/auth_local.plugin.php

```diff
@@ -29,7 +29,6 @@
  * @copyright   2010-2014 Poweradmin Development Team
  * @license     http://opensource.org/licenses/GPL-3.0 GPL
  */
-require_once dirname(dirname(dirname(__DIR__))) . '/vendor/poweradmin/Password.php';
```

Comment:
I've checked that it's working. I included the autoloader provided by composer inside the toolkit.inc.php